geoips/commandline/geoips_get.py

```diff
@@ -3,34 +3,35 @@
 Retrieves the appropriate family/interface/package/plugin based on the args provided.
 """
 
-from importlib import resources, import_module
+from importlib import metadata, resources, import_module
 
 import yaml
 
 from geoips.commandline.geoips_command import GeoipsCommand, GeoipsExecutableCommand
+from geoips.create_plugin_registries import format_docstring
```

Comment:
This seems like an odd location to be importing from in this module. I think a refactor is probably needed here. We can create an issue for it, but I think we need to examine why this is required and fix the root of the problem.

Reply:
I'll create an issue for this. I agree that we should talk about this more. This function was addressing #572.

Reply:
See #630.